src/main/resources/com/veertu/plugin/anka/AnkaMgmtCloud/config.jelly

```diff
@@ -28,42 +28,42 @@
         </f:entry>
-        <f:entry title="${%Launch Delay In Seconds}" field="sshLaunchDelaySeconds" description="The number of seconds to wait before starting launch">
+        <f:entry title="${%Launch Delay (seconds)}" field="sshLaunchDelaySeconds" description="The number of seconds to wait before starting the launch process">
                 <f:textbox clazz="number" default="15"/>
         </f:entry>
-        <f:entry title="${%Monitor recurrence}" field="monitorRecurrenceMinutes" description="Number of minutes to check for dead instances (global). min 1 minute">
+        <f:entry title="${%Monitor Recurrence (minute)}" field="monitorRecurrenceMinutes" description="Number of minutes to check for dead instances (global). min 1 minute">
             <f:textbox clazz="number" default="10"/>
         </f:entry>
-        <f:entry title="${%Connection keep alive seconds}" field="connectionKeepAliveSeconds" description="Number of seconds to keep tcp connection alive">
+        <f:entry title="${%Connection Keep Alive (seconds)}" field="connectionKeepAliveSeconds" description="Number of seconds to keep tcp connection alive">
             <f:textbox clazz="number" default="120"/>
         </f:entry>
-        <f:entry title="${%Max connections}" field="maxConnections" description="Limit for http connection (for the controller)">
+        <f:entry title="${%Maximum HTTP Connections}" field="maxConnections" description="Limit for http connection (for the controller)">
             <f:textbox clazz="number" default="50"/>
         </f:entry>
-        <f:entry title="${%VM Poll time}" field="vmPollTime" description="Time in milliseconds to poll during VM creation">
+        <f:entry title="${%VM Poll Time (milliseconds)}" field="vmPollTime" description="Time in milliseconds to poll during VM creation">
             <f:textbox clazz="number" default="5000"/>
         </f:entry>
 
     </f:advanced>
-    <f:section title="VM templates">
+    <f:section title="Jenkins Node/Agent Templates and Labels">
+        <div>Node/Agent Templates are used for creating Jenkins Agents/Nodes. They differ from the Anka VM Templates, though, they to target specific Anka VM Templates and Tags.</div>
         <!-- templates section-->
         <f:block>
             <j:set var="cloud" value="${instance}"/>
                 <j:if test="${!(cloud!=null)}">
                     <b>
-                        <div>Templates will appear here if the cloud is saved.</div>
-                        <div>You should configure at least one host before you create templates</div>
-                        <div>After configuring hosts click apply and refresh</div>
+                        <div>Jenkins Node/Agent Templates and Labels will appear here if the cloud is reachable (ensure the Anka Build Cloud hostname or IP is reachable from Jenkins).</div>
+                        <div>After configuring the Anka Build Cloud, click apply and refresh.</div>
```

Comment:
Add warning for using Save, so that label's configuration do not get over-written